### PR TITLE
Feature/roleconfig

### LINKED ISSUE #50: Implement roleconfig trigger

### PR DESCRIPTION
**Pull request by Lauraducky**
Fixes #50

### CODE REVIEW

File: src/PockyBot.NET/Constants/Actions.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace PockyBot.NET.Constants
+{
+    class Actions
+    {
+        public const string Get = "get";
+        public const string Set = "set";
+        public const string Delete = "delete";
```
Comment (jimcullenaus) [inline src/PockyBot.NET/Constants/Actions.cs]: I'm confused about this. Does it not already exist? I have it on my branch, and intellisense says it was created by Laura.

Comment (jimcullenaus) [reply src/PockyBot.NET/Constants/Actions.cs]: Side note, why not declare the class `internal static`?

Comment (Lauraducky) [reply src/PockyBot.NET/Constants/Actions.cs]: It doesn't exist on master yet. I'll handle conflicts

Comment (Evangelinexx) [reply src/PockyBot.NET/Constants/Actions.cs]: ConfigActions exists on master?

File: src/PockyBot.NET/Services/Triggers/RoleConfig.cs [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+            if (users.Count == 0)
+            {
+                return "No roles have been set.";
+            }
+
+            var roleConfigMessageBuilder = new StringBuilder("Here is the current config:\n\n");
+            foreach (var user in users)
+            {
+                roleConfigMessageBuilder.Append(
+                    $"- {user.Username}: {string.Join(", ", user.Roles.Select(x => x.UserRole))}\n");
```
Comment (jimcullenaus) [inline src/PockyBot.NET/Services/Triggers/RoleConfig.cs]: For consistency, why not use asterisk for markdown bullet points?

Comment (jimcullenaus) [inline src/PockyBot.NET/Services/Triggers/RoleConfig.cs]: Prefer AppendLine over Append with a manually-added newline, and adjust tests to ensure only testing in a system-agnostic way as much as possible.

**Pull request merged**